cmd/hypervisor/commands/root.go

```diff
@@ -49,49 +48,26 @@ var rootCmd = &cobra.Command{
 			log.Printf("Failed to output build info: %v", err)
 		}
 
+		// Prepare config.
```

Comment:
I would split this function to multiple to make it less complex to read.

Reply:
@nkryuchkov I'm sorry I don't fully agree. In my opinion, the steps of operation are made clear in comments:
```golang
// Prepare config.

// Prepare hypervisor.

// Mock mode.
/* ... or ... */
// Prepare dmsg client.

// Serve HTTP.
```
Is that okay?

Reply:
The comments are great, however I think they show places where functions should be extracted

Reply:
Okay fine.